extensions/interactions/LogicProof/directives/logic-proof-interaction.directive.html

```diff
@@ -4,7 +4,6 @@
     <oppia-codemirror (on-load)="$ctrl.codeEditor($event)"
                       [read-only]="!$ctrl.interactionIsActive">
     </oppia-codemirror>
-    <textarea ng-trim="false" rows="10"></textarea>
```

Comment:
@vojtechjelinek This tag was supposed to removed in #12639. I missed it. @ashutoshc8101 found this, so I have asked him to remove it in this PR.